frontend/src/app/app.tsx

```diff
@@ -38,10 +39,10 @@ const App: React.FC = () => {
             }
         };
 
-        void refreshUser().finally(() => setIsRefreshing(false));
+        void refreshUser().finally(() => setInitialRefreshing(false));
     }, [dispatch]);
 
-    if (isRefreshing) {
+    if (initialRefreshing || isRefreshing) {
```

Comment:
Why do we need two states for this refresh?

Reply:
Before refreshUser() is executed in useEffect, there's an issue that landing page or auth is displayed